### ISSUE TITLE
Move kernel_exec_on_cell into a separate npm module

### ISSUE THREAD

**Issue by Madhu94** [ipython-contrib/jupyter_contrib_nbextensions#1263]
kernel_exec_on_cell.js seems very useful, would it be possible to publish this as a named amd module to npm ?

I would like to build a custom extension on top of this, but I'd also like to be able to use it in isolation -
i.e. I want people to be able to use my extension with/without installing jupyter_contrib_nbextensions.

**Comment by jcb91** [comment]
err, yeah, I don't see why not. You can include your own copy as  per the project's bsd license, but I guess it would  be better update-wise to have a proper package. I haven't published anything on npm before, but sure, why not

**Comment by Madhu94** [comment]
Yes, I was thinking that updating and consuming might be easier with an npm package. Please do this if you think this might be useful for other people - this is a great package that would make building any kind of formatting/linting extensions in the notebook easier.

**Comment by Madhu94** [comment]
Hi, any updates ? Let me know if there is anything I can do to help here.

**Comment by jcb91** [comment]
Done! See https://www.npmjs.com/package/jupyter-notebook-kernel-exec-on-cells. I've done this from a branch on my fork of the code_prettify repo (https://github.com/jcb91/code_prettify/tree/npm). Feel free to make PRs etc to that branch, or however makes most sense to you...

**Comment by jcb91** [comment]
closing as (hopefully!) fixed - feel free to reopen as required :)

**Comment by Madhu94** [comment]
Thank you so much !
